### PR TITLE
feat: basic support for tables in wysiwym

### PR DESCRIPTION
**Pull request by jpuri**
Fix https://github.com/tinacms/tinacms/issues/549#issuecomment-572045203
  
Screenshot 2020-01-13 at 7 51 13 PM

### CODE REVIEW

**Comment by jpuri** [comment]
@spbyrne : basic table feature still needs improvements in style:
1. Table icons to be used in top menu bar.
2. Details for table header and cell styles.

File: packages/@tinacms/fields/stories/wysiwyg.stories.tsx [hunk 1]
```diff
@@ -52,6 +52,27 @@ const WithImage = () => {
   )
 }
 
+const WithTable = () => {
+  const [value, setValue] = React.useState(
+    `| A | B | C |
+| --- | --- | ---:|
+| aaa | bbb | ccc |
```
Comment (DirtyF) [inline packages/@tinacms/fields/stories/wysiwyg.stories.tsx]: Can we test with some Makdown inside cells? (italic, bold, list?)

**Comment by jpuri** [comment]
@DirtyF : I found that though wysiwym support showing different text formatting in table:

Screenshot 2020-01-14 at 11 02 15 AM

The markdown generated by table is not correct, I will create a separate github issue to address this.

**Comment by jamespohalloran** [comment]
Very cool!!

**Comment by jpuri** [comment]
Hey @spbyrne : kets try to merge this soon. More work on tables we can continue on following PRs.

**Comment by ncphillips** [comment]
Great! I'll get this merged today 😄

**Pull request merged**